src/Ranking.elm

```diff
@@ -23,60 +23,71 @@ main =
 
 
 type alias Model =
-    { teams : Zipper (Zipper Team)
-    , isShowingInfo : Bool -- What I need to do with this variable?
+    { teams : Zipper Team
+    , isShowingInfo : Bool
     , inPage : Int
-    , windowHeight : Int
     }
 
 
 type Msg
     = NextPage
 
 
-type alias Team =
-    { number : Int
-    , name : String
-    , score : Float
-    }
-
-
-moveZipperBy : Int -> Zipper a -> Zipper a -- Never failed
+moveZipperBy : Int -> Zipper a -> Zipper a
 moveZipperBy moveBy zipper =
-    let
-        zipperNextCurrent : Maybe (Zipper a)
-        zipperNextCurrent = next zipper
-    in
-        if moveBy == 0 || zipperNextCurrent == Nothing then
-            zipper
-        else
-            next zipper
-            moveZipperBy (moveBy - 1) zipper
+    if moveBy == 0 || isLast zipper then
+        zipper
+
+    else
+        moveZipperBy (moveBy - 1) <| withDefault (current zipper) <| next zipper
```

Comment:
I think it would be better here to let the user of this function know if the function failed. I think it will give more information.